src/transformers/generation_utils.py

```diff
@@ -904,6 +905,11 @@ def generate(
                 List of token ids that are not allowed to be generated. In order to get the token ids of the words that
                 should not appear in the generated text, use `tokenizer(bad_words, add_prefix_space=True,
                 add_special_tokens=False).input_ids`.
+            force_words_ids(`List[List[int]]` or `List[List[List[int]]]`, *optional*):
```

Comment:
Think this is the only thing that we should maybe discuss a bit in the PR. Would it maybe be easier to understand for users to provide two function args instead: one `force_word_ids` -> IMO this really corresponds to the model being forced to generate exactly those words so not the `disjunctive constraint` case. Would it maybe cleaner to in addition also add a `force_one_of_word_list_ids` arg for disjunctive constraint decoding (or a better naming)? What do you think @cwkeam ?